src/forms/initialize-dispense-form-from-requests.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I think i might need some more context on how this will be different from using the dispense-form component

Reply:
I just added some more comments above that hopefully help to explain it.

This component extracts out from the DispenseForm the logic that initialized the dispenses payload from the existing medication.  This allows DispenseForm to be used for both creating new medication dispenses and editing existing ones.